kibbeh/src/ui/FollowersOnline.tsx

```diff
@@ -21,7 +21,7 @@ export interface FriendsOnlineProps {
   showMoreAction?: MouseEventHandler<HTMLDivElement>;
 }
 
-export const FollowerOnline: React.FC<UserWithFollowInfo> = ({
+export const FollowersOnline: React.FC<UserWithFollowInfo> = ({
```

Comment:
This component should just stay as `FollowerOnline` as it isn't a list of followers, rather it is a component containing a **single** follower's data *(username, avatar, etc)* and links to their profile and room they're in

Reply:
also no to forget. you're not changing any imports in any other files. this will end up break the code.

Reply:
> also no to forget. you're not changing any imports in any other files. this will end up break the code.

yeah, you're right. if you look at [kibbeh/src/stories/FriendsOnline.story.tsx](https://github.com/benawad/dogehouse/blob/a53dba92ecda6ec7f182a781edd3127e57a53537/kibbeh/src/stories/FriendsOnline.story.tsx).
it is begin imported as FollowersOnline. it is causing errors. that's the line that should be fixed

Reply:
also, is correct to name this component as FollowersOnline when the export is FollowerOnline?.